plugins/Ecommerce/tests/System/expected/test_ecommerceOrderWithItemsmultipleDates_andMultipleWebsites__Goals.getItemsCategory_day.xml

```diff
@@ -7,53 +7,89 @@
 				<revenue>2500</revenue>
 				<quantity>3</quantity>
 				<orders>2</orders>
+				<nb_visits>3</nb_visits>
```

Comment:
The product view metrics were missing here before because of 
https://github.com/matomo-org/matomo/blob/bc499def7f848983dc85ced1373d7f6c0b41e38d/plugins/Goals/API.php#L792-L796
That's no longer the case, as product views are directly archived...